dispatcher/service/pool.py

```diff
@@ -73,6 +74,18 @@ def cancel(self) -> None:
         self.is_active_cancel = True  # signal for result callback
         self.process.terminate()  # SIGTERM
 
+    def get_data(self) -> dict[str, Any]:
+        return {
+            'worker_id': self.worker_id,
+            'pid': self.process.pid,
+            'status': self.status,
+            'finished_count': 0,
```

Comment:
what does this indicate and why is it hardcoded to 0? I didn't see any other place in the code where this is updated.